dojo/api_v2/views.py

```diff
@@ -2945,7 +2945,7 @@ def report_generate(request, obj, options):
         )
         report_name = "Engagement Report: " + str(engagement)
 
-        ids = set(finding.id for finding in findings.qs)
+        ids = {finding.id for finding in findings.qs}
```

Comment:
Switching to this form means you have to look for a `:` in the comprehension to determine whether it's producing a `set` or a `dict`. Speaking as someone who no longer writes Python every day, I don't think this rule (`C401`) actually increases code clarity and I feel like we should disable it.

Reply:
I agree with cneil on this one - seems to make code harder to understand at the expense of a couple of characters. That feels like a bad trade-off to me.